tests/static-check/run_checks.sh

```diff
@@ -3,6 +3,7 @@
 set -x
 
 n_procs="$(getconf _NPROCESSORS_ONLN)"
+use_procs=$((n_procs/2))
```

Comment:
@craigcomstock what happens if `n_procs` is 1?

I guess 0 means nothing in parallel, 1 job at a time?

man page;

```
       -l [load], --load-average[=load]
            Specifies that no new jobs (commands) should be started if there are others jobs running and the load average is at least load (a floating-point number).
            With no argument, removes a previous load limit.
```